### PR TITLE
Update performance script to python 3, and minor corrections

### PR DESCRIPTION
**Pull request by mark-petersen**
Minor script updates. With these changes, the script produces performance scaling plots correctly.

### CODE REVIEW

File: ocean/performance_testing/performance_test.py [hunk 1]
```diff
@@ -165,6 +165,6 @@
     figurenamepath = "figures_performance/" + resolution_name + '_' + \
         machine_long_name + '_' + str(nprocs_max) + "_" + timenow + ".png"
     plt.savefig(figurenamepath)
-    i = i / 2
+    i = int( i / 2 )
```
Comment (xylar) [inline ocean/performance_testing/performance_test.py:168]: The preferred way to do this is:
```suggestion
    i = i // 2
```

**Comment by mark-petersen** [comment]
In a run directory, the command
```
./performance_test.py 1024 32 grizzly 36 EC60to30
```
produces this plot:
EC60to30_grizzly_1024_20210719_081846

**Comment by xylar** [comment]
@mark-petersen, if you want to make the one suggested change above, I'll approve by inspection. If you would like me to test, please let me know what typical usage looks like.

**Comment by xylar** [comment]
@mark-petersen, based on your typical usage above (our messages crossed!), would you be willing to add `argparse` to this script so the arguments are named, rather than ordered and so there's help when you do `./performance_test.py  --help`.

Should we add this to the conda package?

**Comment by xylar** [comment]
If the answer is no to all of the above, that's fine.

**Comment by mark-petersen** [comment]
@xylar I agree about argparse. I had thought of that too. If you approve I'll merge this in now so students can use it, and argparse is a good exercise for a student to update.

**Comment by mark-petersen** [comment]
@xylar no need to test, visual inspection is fine.

**Review by xylar — APPROVED** [review]
Looks good.

**Pull request merged**